### ISSUE TITLE
New functions

### ISSUE THREAD

**Issue by PeterTillema** [PeterTillema/ICE#70]
LEFT( for bitshift left
RIGHT( for rightshift
break?
continue?
ElseIf (instead of Else:If)

**Comment by Ti64CLi** [comment]
You didn't add the RIGHT( function?

**Comment by PeterTillema** [comment]
https://github.com/PeterTillema/ICE/blob/9a0a180f8d50643e6270e10dcf02789b7db5ac97/src/functions.c#L660-L707

**Comment by Ti64CLi** [comment]
Oh, ok, sorry for that. So, I guess LEFT is also implemented?

**Comment by Ti64CLi** [comment]
It's so weird.
I saw that LEFT wasn't implemented yet
`// LEFT(
            if (function2 == tLEFT) {
}`
But he works on my calc ^^'

**Comment by Ti64CLi** [comment]
Though, when I compile that code :
Disp LEFT(4,1)
It displays 8

**Comment by PeterTillema** [comment]
Yes, because both arguments are numbers, and then it gets optimized:
https://github.com/PeterTillema/ICE/blob/a54747033004cfff9ae81a776fe332420d419908/src/parse.c#L666-L667

**Comment by Ti64CLi** [comment]
Ok, thank you

**Comment by Xenoxis** [comment]
`Wait`, `Stop`, `DelVar`, `Menu(` can be implemented ?
And if yes, will they be ?

**Comment by Xenoxis** [comment]
Will also be good if you can add `switch` statement with `case`

**Comment by Xenoxis** [comment]
Is it bad if it looks like C ? Because `switch` can be interesting for some variables processing as interger and would be more efficient than lots of if, isen't it ?

**Comment by Ti64CLi** [comment]
I agree with Xenoxis, Menu( could be very useful ;)

**Comment by LAX18** [comment]
I would argue against it, as it would be used on the main screen, not in the canvas mode. I, personally, do not see many ICE programs not use the drawing canvas, as ICE's main attraction is the use of the canvas to expand upon BASIC programs.

**Comment by PeterTillema** [comment]
Sure, `Menu(` is very useful, but it would be very large code, and you can 'easily' write it yourself.
